src/components/Onfido/index.native.js

```diff
@@ -56,7 +56,7 @@ class Onfido extends React.Component {
                     errorMessage,
                 )) {
                     Alert.alert(
-                        this.props.translate('onfidoStep.cameraRequestTitle'),
+                        this.props.translate('cameraPermissionsNotGranted'),
```

Comment:
Could we please move the `cameraPermissionsNotGranted` to the `onfidoStep`? It is not used anywhere else and it is uncommon to have the keys in the first level.